fuzz_lightyear/main.py

```diff
@@ -67,10 +68,11 @@ def setup_client(
         return 'Unable to connect to server.'
     except (
         simplejson.errors.JSONDecodeError,      # type: ignore
+        yaml.YAMLError,
```

Comment:
Will `mypy` complain about no `# type: ignore` on the `YAMLError` line?

Reply:
Yep, `mypy` runs in `make test` which has passed. I think the package structure for `simplejson` is snowflakey which is why we needed the `#type: ignore` for that line.